packages/fxa-content-server/app/scripts/lib/config-loader.js

```diff
@@ -31,15 +30,15 @@ ConfigLoader.prototype = {
     return this._readConfigFromHTML()
       .then(this._parseHTMLConfig)
       .then((config) => {
-        config.lang = $('html').attr('lang');
+        config.lang = document.querySelector('html').getAttribute('lang');
```

Comment:
This is an orthogonal change to see if I could get rid of the jQuery dependency in the subscription page.

Reply:
I put this into #939